src/pages/newPost/NewPost.jsx

```diff
@@ -0,0 +1,128 @@
+            console.error(e);
+            toggleErrorAddPost(true);
+        }
+    }
+
+    return (
+        <main>
+            <h2>Verstuur hier een nieuwe post</h2>
+            <form onSubmit={handleFormSubmit}>
+                {errorAddPost &&
+                    <p className="error-message">Er gaat helaas iets mis, probeer het later nog eens ...</p>}
+
+                <NewPostForm
+                    inputType="text"
+                    inputName="title"
+                    inputLabel="Titel:"
+                    inputId="title"
+                    validationRules={{
+                        required: {
+                            value: true,
+                            message: "Titel veld is verplicht",
+                        },
+                        maxLength: {
+                            value: 20,
+                            message: "Titel mag maximaal 20 karakters zijn",
+                        },
+                    }}
+                    register={register}
+                    errors={errors}
+                />
+
+                <NewPostForm
+                    inputType="text"
+                    inputName="subtitle"
+                    inputLabel="Subtitel:"
+                    inputId="subtitle"
+                    validationRules={{
+                        required: {
+                            value: true,
+                            message: "Subtitel veld is verplicht",
+                        },
+                        maxLength: {
+                            value: 30,
+                            message: "Subtitel mag maximaal 30 karakters omvatten",
+                        },
+                    }}
+                    register={register}
+                    errors={errors}
+                />
+
+                <NewPostForm
+                    inputType="text"
+                    inputName="author"
+                    inputLabel="Auteur:"
+                    inputId="author"
+                    validationRules={{
+                        required: {
+                            value: true,
+                            message: "Auteur veld is verplicht",
+                        },
+                        maxLength: {
+                            value: 30,
+                            message: "Auteur mag maximaal 30 karakters zijn",
+                        },
+                    }}
+                    register={register}
+                    errors={errors}
+                />
+
+                <label htmlFor="post-content">Blog bericht:</label>
+                <textarea
+                    name="content"
+                    id="blog-content"
+                    cols="30"
+                    rows="10"
+                    required
+                    minLength={300}
+                    maxLength={2000}>
+                </textarea>
+
+                <button type="submit">Versturen</button>
+                {Object.keys(addPost).length > 0 &&
```

Comment:
Hier wordt die dan makkelijker
```suggestion
                {addPost &&
```